BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs

```diff
@@ -8,11 +16,38 @@ namespace BackendApartmentReservation.Controllers
     [ApiController]
     public class ProfilesController : ControllerBase
     {
+        private readonly UserManager<DbEmployee> _userManager;
         [HttpGet]
         [Route("profiles")]
         public async Task<IEnumerable<string>> Get()
         {
             return await Task.FromResult(new[] { "userProfile1", "userProfile2" });
         }
+
+        [HttpPost]
+        [AllowAnonymous]
+        [Route("register")]
+        public async Task<IActionResult> Register([FromBody] RegisterRequest model)
+        {
+            if (!ModelState.IsValid) return BadRequest(ModelState);
+            if (RegistrationInputValidator.IsInputValid(model)) return BadRequest("Input is not valid");
+
+            var employee = new DbEmployee
+            {
+                FirstName = model.FirstName,
+                LastName = model.LastName,
+                Email = model.Email,
+                Office = model.Office
+            };
+
+            var result = await _userManager.CreateAsync(employee, model.Password);
```

Comment:
One more thing. Controller should pretty much have no logic at all except calling manager and potentially try-catching exceptions of this manager, if any. But this is more of a future notice, since that is probably going to change when we introduce manager

Reply:
I'll keep it here for now, but a valid point